index.html

```diff
@@ -1658,14 +1700,27 @@
 	}
 
 	function shuffle(arr, num) {
-		if(num == undefined)
-			num = arr.length;
+		if(num == undefined) num = arr.length;
+		
+	/*
 		for(var i = 0; i < num; i++) {
 			var j = Math.floor(Math.random() * (arr.length - i - 1)) + i + 1;
 			var temp = arr[i];
 			arr[i] = arr[j];
 			arr[j] = temp;
 		}
+		*/
```

Comment:
It looks like it was a simple off-by-1 error, easy enough to correct without rewriting everything!